README.md

```diff
@@ -36,10 +36,11 @@
 
```

Comment:
Can we align the introductory paragraph with other SDKs, using the [Readme guidelines](https://www.notion.so/fingerprintjs/Writing-DX-GitHub-Readmes-9556ed587c0749b29f32a18017413cbc?pvs=4#9c47b1318fa647d6af91254004bdb9b8)?

Reply:
Sure, I'll work on it 👍